package.json

```diff
@@ -16,6 +16,7 @@
     "dist/index.d.ts"
   ],
   "dependencies": {
+    "@types/node": "^15.6.1",
```

Comment:
Needs to be a prod dependency?

Reply:
Yeah - this needs to be installed for `mem-fs`es type-declaration to be intact.

That confused me at first, too, tbh 😂

Reply:
Oh - I just noticed, that `@types/vinyl` is being used in the type-declaration, too, and is missing from the `dependencies`-array - I'll fix it in this PR as well